Firestore/core/test/firebase/firestore/local/lru_garbage_collector_test.cc

```diff
@@ -448,29 +452,32 @@ TEST_P(LruGarbageCollectorTest, RemoveOrphanedDocuments) {
           << "Missing document " << key.ToString().c_str();
     }
   });
-  persistence_->Shutdown();
 }
 
 // TODO(gsoltis): write a test that includes limbo documents
 
 TEST_P(LruGarbageCollectorTest, RemoveTargetsThenGC) {
-  // Create 3 targets, add docs to all of them
-  // Leave oldest target alone, it is still live
-  // Remove newest target
-  // Blind write 2 documents
-  // Add one of the blind write docs to oldest target (preserves it)
-  // Remove some documents from middle target (bumps sequence number)
-  // Add some documents from newest target to oldest target (preserves them)
-  // Update a doc from middle target
-  // Remove middle target
-  // Do a blind write
-  // GC up to but not including the removal of the middle target
+  // Setup:
+  //   - Create 3 targets, add docs to all of them.
+  //   - Leave oldest target alone, it is still alive.
+  //   - Remove newest target.
+  //   - Blind write 2 documents.
+  //   - Add one of the blind write docs to oldest target (preserves it).
+  //   - Remove some documents from middle target (bumps sequence number).
+  //   - Add some documents from newest target to oldest target (preserves
+  //   - them).
+  //   - Update a doc from middle target.
+  //   - Remove middle target.
+  //   - Do a blind write.
+  //   - GC up to but not including the removal of the middle target.
   //
   // Expect:
-  // All docs in oldest target are still around
-  // One blind write is gone, the first one not added to oldest target
-  // Documents removed from middle target are gone, except ones added to oldest
-  // target Documents from newest target are gone, except
+  //   - All docs in oldest target are still around.
+  //   - One blind write is gone, the first one not added to oldest target.
```

Comment:
Ultranit: s/oldest/the oldest (here and below).

Reply:
Done.